src/Model/Solid/Finite/Adhesion.H

```diff
@@ -0,0 +1,225 @@
+
```

Comment:
Please add documentation at the top of this file to outline the method. See `Model::Solid::Affine::J2` for an example of suitable documentation.